docs/source/contributing/documentation.rst

```diff
@@ -3,8 +3,80 @@ Adding Documentation
 ====================
 
 When submitting a new class through a PR, or any changes in general,
-there should be documentation where possible. Here are the guidelines
-for writing it.
+there should be documentation where possible. Here are our guidelines
+for writing documentation.
+
+Guidelines for examples
+-----------------------
+
+Everybody is welcome to contribute examples to the documentation. Since straightforward examples are a great resource for quickly learning manim, here are some guidelines.
+
+What makes a great example
+--------------------------
+
+.. note:: 
+
+   As soon as a new version of manim is released, the documentation will be a snapshot of that version. Examples contributed after the release will only be shown in the latest documentation.
+   
+* Examples should be ready to copy and paste for use.
+
+* Examples should be brief yet still easy to understand.
+
+* Examples don't require the ``from manim import *`` statement, this will be added automatically when the docs are build```
+
+* There should be a balance of animated and non-animated examples.
+
+- As manim makes animations, we can include lots of animated examples; however, our RTD has a maximum 15 minutes to build. Animated examples should only be used when necessary, as last frame examples render faster.
+
+- Lots of examples (e.g. size of a plot-axis, setting opacities, making texts, etc.) will also work as images. It is a lot more convenient to see the end product immediately instead of waiting for an animation to reveal it.
+
+* Examples should not have comments in the code. Explanations should be added outside of the example code.
```

Comment:
I don't necessarily agree with this; I think code-comments make sense as well.

Reply:
I was thinking about that too, but I just copied it to make sure that you people won't find any difficulties. I will be removing line 33 then..